spec/acceptance/pe_metric_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper_acceptance'
+
+describe 'test default and system includes' do
+  before(:all) do
+    pp = <<-MANIFEST
+        include puppet_metrics_collector
+        include puppet_metrics_collector::system
+        MANIFEST
+    idempotent_apply(pp)
+  end
+  it 'checks for pe_metric services' do
+    run_shell('systemctl list-units --type=service | grep "metric"') do |r|
```

Comment:
The following regex aligns with the naming scheme to ensure there are no false positives. 

```
systemctl list-units --type=service |egrep '^puppet_[^ ]*metrics'
```